CoreUtilFramework/Network/APIResponse.swift

```diff
@@ -24,6 +24,10 @@ public struct ApiError {
         }
         return "ApiError"
     }
+    public func messageWithErrorCode(messageTemplate:String = "%@ (%@)")->String?{
+        guard let message = message, let code = code else { return nil }
+        return String(format: messageTemplate, message, code)
+    }
```

Comment:
Spacing düzeltilebilir.  😇
```suggestion

    public func messageWithErrorCode(messageTemplate: String = "%@ (%@)") -> String? {
        guard let message = message, let code = code else { return nil }
        return String(format: messageTemplate, message, code)
    }
```